cmd/channel/email/main.go

```diff
@@ -13,9 +17,11 @@ import (
 )
 
 type Email struct {
-	Host string `json:"host"`
-	Port string `json:"port"`
-	From string `json:"from"`
+	Host     string `json:"host"`
+	Port     string `json:"port"`
+	From     string `json:"from"`
+	Password string `json:"password"`
+	UseTLS   bool   `json:"tls"`
```

Comment:
I'm not sure if this is the right place to discuss this, but SMTP transport encryption is more ternary than binary.
There is:
- plaintext: no transport encryption at all
- STARTTLS: start with a plaintext SMTP connection and try to upgrade it to TLS (with all possible issues, <https://nostarttls.secvuln.info/>)
- TLS: TLS tunnel with SMTP within

By both calling the variable `UseTLS` and also using `tls` as the element's name, I find this misleading as a STARTTLS connection will be established.
https://github.com/Icinga/icinga-notifications/blob/64067793a9fccc42498051561eb91d78ea55638c/cmd/channel/email/main.go#L159-L166
https://github.com/Icinga/icinga-notifications/blob/64067793a9fccc42498051561eb91d78ea55638c/cmd/channel/email/main.go#L55-L59